checkers/flags_1_12.go

```diff
@@ -0,0 +1,11 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+// +build go1.12
+// +build !go1.13,!go1.14
+
+package checkers
+
+const (
+	flagRO flag = 1<<5 | 1<<6
```

Comment:
Should all three of these build files be identical?

The original looked like it had `1<<0` as one of the flags

Reply:
I guess I just wanted a file per go version we support. The 1<<0 flag was for go 1.3. Since this is go mod, we don't support go < 1.12